### PR TITLE
Update release changes from 1.5.3

### PR DESCRIPTION
**Pull request by daveneiman**
There are a couple of changes that are in addition to log4j2:
- An <scm> stanza change in the POM
- An dependency addition to avoid an exception when running FITS in Java 8

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,9 +14,9 @@
 		<java.version>1.8</java.version>
 		<java.target>1.8</java.target>
 		<bcmail.version>132</bcmail.version>
-		<log4j.version>1.2.17</log4j.version>
-		<slf4j.version>1.7.22</slf4j.version>
-		<tika.version>1.22</tika.version>
+		<log4j.version>2.17.2</log4j.version>
+		<slf4j.version>1.7.36</slf4j.version>
+		<tika.version>2.3.0</tika.version>
```
[the remark this answers is not in the document]
Comment (daveneiman) [reply pom.xml:19]: This release is strictly the upgrade to log4j2 that @awoods did via a cherrypick.

[the remark this answers is not in the document]
Comment (pwinckles) [reply pom.xml:19]: It looks like the 1.5.3 release was made referencing Tika 2 in the pom and Tika 1 in the lib directory. I am very surprised that the tests didn't blow up

[the remark this answers is not in the document]
Comment (daveneiman) [reply pom.xml:19]: Agreed. This release is so we could resolve the log4j security issue so we could deploy within the Harvard ecosystem before figuring out how to resolve the codec issue that was raised by Preservation Services.

[the remark this answers is not in the document]
Comment (daveneiman) [reply pom.xml:19]: The tests all passed as did running this FITS build on files.

[the remark this answers is not in the document]
Comment (pwinckles) [reply pom.xml:19]: But, does it actually work? There were a lot changes from Tika 1 to 2

[the remark this answers is not in the document]
Comment (pwinckles) [reply pom.xml:19]: Sorry, didn't see your followup when I posted

**Pull request closed without merging**